src/views/AnnualReport.vue

```diff
@@ -608,36 +608,6 @@ export default class AnnualReport extends Mixins(
     this.$root.$off('fetch-error-event')
   }
 
-  beforeRouteLeave (to, from, next): void {
```

Comment:
I didn't see this lifecycle event being called. It used to work. 🤷‍♂️ 

This code is now in `goToDashboard()`.